users/tests/test_utils_functions.py

```diff
@@ -1,60 +1,56 @@
+from django.core.exceptions import ValidationError
 from django.test import TestCase
-from rest_framework import serializers
 
 import users.common.utils
 import users.common.validators
-from users.common.exceptions import CustomValidationError
 from users.common.strings import CustomValidationErrorText
-from users.serializers import CustomRegisterSerializer
 
 
 class TestCustomModelEmailValidationFunction(TestCase):
     def test_user_email_should_hold_at_least_one_at_sign(self):
```

Comment:
You should use `parametrized` for test methods in this class as they are not different from tests in `TestCustomSerializerEmailValidationFunction` class. Check here https://github.com/Code-Poets/sheetstorm/blob/master/managers/tests/test_views.py#L117.

Reply:
I am not sure if I understood, because I have never used the parameterized test before, but I tried to implement it and it is ready to look at it, now :).

Reply:
Oh wow, I was rather thinking about combining this class and `TestCustomSerializerEmailValidationFunction` but you take a different way and its nice too. But please take a loon on tests in `TestCustomSerializerEmailValidationFunction` - doesn't they do exactly the same now?

Reply:
Yes they are exactly the same, so should I delete those test ? We will not use serializers any more and what is more tests looks the same. For now I will remove this class `TestCustomSerializerEmailValidationFunction`

Reply:
Do not remove yet, we will remove it along with serializer, for now lets keep it.